### PR TITLE
Add Spanish translation for hardware documentation

### PR DESCRIPTION
**Pull request by michizhou**
Additional Spanish translation pages. 

## Description
Translates all hardware section documentation files as a supplement to existing translated pages. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* #6924 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by yanfali** [comment]
@michizhou is this Google translate?

**Comment by michizhou** [comment]
@michizhou This isn't Google Translate. I studied Spanish for three years in high school.

**Comment by noroadsleft** [comment]
@namelivia and @kbravh Would you guys mind looking this over for us?

File: docs/es/hardware_avr.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+# Teclados con Procesodores AVR
+
+Esta página describe el soporte para procesadores AVR en QMK. Procesadores AVR incluyen el atmega32u4, atmega32u2, at90usb1286, y otros procesadores de Corporación Atmel. Procesadores AVR son MCU's de 8-bit que están diseñados para ser fáciles de trabajar con. Los procesadores AVR más comunes en teclados tienen USB a bordo y un montón de GPIO para soportar grandes matrices de teclado. Son el MCU más popular para el uso en los teclados de hoy.
```
Comment (kbravh) [inline docs/es/hardware_avr.md]:
```suggestion
Esta página describe el soporte para procesadores AVR en QMK. Los procesadores AVR incluyen el atmega32u4, atmega32u2, at90usb1286, y otros procesadores de la Corporación Atmel. Los procesadores AVR son MCUs de 8-bit que son diseñados para ser fáciles de trabajar. Los procesadores AVR más comunes en los teclados tienen USB y un montón de GPIO para permitir grandes matrices de teclado. Son los MCUs más populares para el uso en los teclados hoy en día.
```

File: docs/es/hardware_avr.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+Si aún no lo hecho debes leer las [Pautas del teclado](hardware_keyboard_guidelines.md) para tener una idea de cómo teclados encajan en QMK.
```
Comment (kbravh) [inline docs/es/hardware_avr.md]:
```suggestion
Si aún no lo has hecho, debes leer las [Pautas de teclados](hardware_keyboard_guidelines.md) para tener una idea de cómo los teclados encajan en QMK.
```

File: docs/es/hardware_avr.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+## Añadir tu Teclado AVR a QMK
+
+QMK tiene una seria de características para simplificar el trabajo con teclados AVR. Para la mayoría de los teclados no tienes que escribir una sola línea de código. Para empezar, ejecuta el script `util/new_keyboard.sh`:
```
Comment (kbravh) [inline docs/es/hardware_avr.md]:
```suggestion
QMK tiene varias características para simplificar el trabajo con teclados AVR. Para la mayoría de los teclados no tienes que escribir ni una sola línea de código. Para empezar, ejecuta el archivo `util/new_keyboard.sh`:
```

File: docs/es/hardware_avr.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+Replacing %KEYBOARD% with mycoolkb... done
+Replacing %YOUR_NAME% with John Smith... done
+
+Created a new keyboard called mycoolkb.
+
+To start working on things, cd into keyboards/mycoolkb,
+or open the directory in your favourite text editor.
+```
+
+Esto creará todos los archivos necesarios para soportar tu nuevo teclado, y rellenar la configuración con valores predeterminados. Ahora sólo tienes que personalizarlo para tu teclado.
```
Comment (kbravh) [inline docs/es/hardware_avr.md]:
```suggestion
Esto creará todos los archivos necesarios para tu nuevo teclado, y rellenará la configuración con valores predeterminados. Ahora sólo tienes que personalizarlo para tu teclado. 
```

File: docs/es/hardware_avr.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+
+To start working on things, cd into keyboards/mycoolkb,
+or open the directory in your favourite text editor.
+```
+
+Esto creará todos los archivos necesarios para soportar tu nuevo teclado, y rellenar la configuración con valores predeterminados. Ahora sólo tienes que personalizarlo para tu teclado. 
+
+## `readme.md`
+
+Aquí está donde describirás tu teclado. Por favor sigas la [Plantilla Readme de Teclado](documentation_templates.md#keyboard-readmemd-template) al escribir tu `readme.md`. Te animamos a colocar una imagen en la parte superior de tu `readme.md`. Por favor, utilices un servicio externo como [Imgur](http://imgur.com) para alojar las imágenes.
```
Comment (kbravh) [inline docs/es/hardware_avr.md]:
```suggestion
Aquí es donde describirás tu teclado. Por favor sigue la [Plantilla del readme de teclados](documentation_templates.md#keyboard-readmemd-template) al escribir tu `readme.md`. Te animamos a colocar una imagen en la parte superior de tu `readme.md`. Por favor, utiliza un servicio externo como [Imgur](http://imgur.com) para alojar las imágenes.
```

File: docs/es/hardware_avr.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+
+## `<keyboard>.c`
+
+Aquí está donde va toda la lógica personalizada para tu teclado. Muchos teclados no necesitan poner nada aquí. Puedes aprender más sobre cómo escribir lógica personalizada en [Funciones Quantum Personalizadas](custom_quantum_functions.md).
```
Comment (kbravh) [inline docs/es/hardware_avr.md]:
```suggestion
Aquí es donde pondrás toda la lógica personalizada para tu teclado. Muchos teclados no necesitan nada aquí. Puedes aprender más sobre cómo escribir lógica personalizada en [Funciones Quantum Personalizadas](custom_quantum_functions.md).
```

File: docs/es/hardware_avr.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+
+## `<keyboard>.h`
+
+Este es el archivo en el que define tu(s) [Macro(s) de Layout](feature_layouts.md). Por lo menos deberías tener un `#define LAYOUT` para tu teclado que se ve algo como esto:
```
Comment (kbravh) [inline docs/es/hardware_avr.md]:
```suggestion
Este es el archivo en el que defines tu(s) [Macro(s) de Layout](feature_layouts.md). Por lo menos deberías tener un `#define LAYOUT` para tu teclado que se ve algo así:
```

File: docs/es/hardware_avr.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+      k10,   k11         \
+) {                      \
+    { k00, k01,   k02 }, \
+    { k10, KC_NO, k11 }, \
+}
+```
+
+La primera mitad de la macro pre-procesador `LAYOUT` define la disposición física de las llaves. La segunda mitad de la macro define la matriz a la que están conectados los interruptores. Esto te permite tener una disposición física de las llaves que difiere de la matriz de cableado.
+
+Cada una de las variables `k__` tiene que ser única, y normalmente siguen el formato `k<row><col>`.
```
Comment (kbravh) [inline docs/es/hardware_avr.md]:
```suggestion
Cada una de las variables `k__` tiene que ser única, y normalmente sigue el formato `k<row><col>`.
```

**Comment by namelivia** [comment]
I'll review it too

File: docs/es/hardware_avr.md [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+
+La matriz física (la segunda mitad) debe tener un número de filas igualando `MATRIX_ROWS`, y cada fila debe tener exactamente `MATRIX_COLS` elementos. Si no tienes tantas teclas físicas puedes usar `KC_NO` para rellenar los espacios en blanco.
+
+## `config.h`
+
+El archivo `config.h` está donde configuras el hardware y el conjunto de características para tu teclado. Hay un montón de opciones que se pueden colocar en ese archivo, demasiados para listar allí. Para obtener una visión de conjunto completa de las opciones disponibles consultes la página de [Opciones de Configuración](config_options.md).
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
El archivo `config.h` es donde configuras el hardware y el conjunto de características para tu teclado. Hay un montón de opciones que se pueden colocar en ese archivo, demasiadas para listar allí. Para obtener una visión de conjunto completa de las opciones disponibles consulta la página de [Opciones de Configuración](config_options.md).
```

File: docs/es/hardware_avr.md [hunk 10]
```diff
@@ -0,0 +1,181 @@
...
+
+### Configuración de hardware
+
+
+En la parte superior de `config.h` encontrarás ajustes relacionados con USB. Estos controlan la apariencia de tu teclado en el Sistema Operativo. Si no tienes una buena razón para cambiar debes dejar el `VENDOR_ID` como `0xFEED`. Para el `PRODUCT_ID` debes seleccionar un número que todavía no está en uso.
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
En la parte superior de `config.h` encontrarás ajustes relacionados con USB. Estos controlan la apariencia de tu teclado en el Sistema Operativo. Si no tienes una buena razón para cambiar debes dejar el `VENDOR_ID` como `0xFEED`. Para el `PRODUCT_ID` debes seleccionar un número que todavía no esté en uso.
```

File: docs/es/hardware_avr.md [hunk 11]
```diff
@@ -0,0 +1,181 @@
...
+
+Cambies las líneas de `MANUFACTURER`, `PRODUCT`, y `DESCRIPTION` para reflejar con precisión tu teclado.
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
Cambia las líneas de `MANUFACTURER`, `PRODUCT`, y `DESCRIPTION` para reflejar con precisión tu teclado.
```

File: docs/es/hardware_avr.md [hunk 12]
```diff
@@ -0,0 +1,181 @@
...
+#define MANUFACTURER    Tú
+#define PRODUCT         mi_teclado_fantastico
+#define DESCRIPTION     Un teclado personalizado
+```
+
+?> Windows y macOS mostrarán el `MANUFACTURER` y `PRODUCT` en la lista de dispositivos USB. `lsusb` en Linux toma estos de la lista mantenida por el [Repositorio de ID USB](http://www.linux-usb.org/usb-ids.html) por defecto. `lsusb -v` mostrará los valores reportados por el dispositivo, y también están presentes en los registros del núcleo después de conectarlo.
+
+### Configuración de la matriz del teclado
+
+La siguiente sección del archivo `config.h` trata de la matriz de tu teclado. Lo primero que debes establecer es el tamaño de la matriz. Esto es generalmente, pero no siempre, el mismo número de filas y columnas como la disposición de la llave física.
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
La siguiente sección del archivo `config.h` trata de la matriz de tu teclado. Lo primero que debes establecer es el tamaño de la matriz. Esto es generalmente, pero no siempre, el mismo número de filas y columnas como la disposición física de  las teclas
```

File: docs/es/hardware_avr.md [hunk 13]
```diff
@@ -0,0 +1,181 @@
...
+
+```c
+#define MATRIX_ROWS 2
+#define MATRIX_COLS 3
+```
+
+Una vez que hayas definido el tamaño de tu matriz, necesitas definir qué patas en tu MCU están conectados a filas y columnas. Para hacerlo simplemente especifiques los nombres de esas patas:
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
Una vez que hayas definido el tamaño de tu matriz, necesitas definir qué pines en tu MCU están conectados a filas y columnas. Para hacerlo simplemente especifica los nombres de esos pines:
```

File: docs/es/hardware_avr.md [hunk 14]
```diff
@@ -0,0 +1,181 @@
...
+
+```c
+#define MATRIX_ROW_PINS { D0, D5 }
+#define MATRIX_COL_PINS { F1, F0, B0 }
+#define UNUSED_PINS
+```
+
+El número de entradas debe ser el mismo que el número que asignaste a `MATRIX_ROWS`, y del mismo modo para `MATRIX_COL_PINS` y `MATRIX_COLS`. No tienes que especificar `UNUSED_PINS`, pero puedes si deseas documentar qué patas están abiertos.
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
El número de entradas debe ser el mismo que el número que asignaste a `MATRIX_ROWS`, y del mismo modo para `MATRIX_COL_PINS` y `MATRIX_COLS`. No tienes que especificar `UNUSED_PINS`, pero puedes si deseas documentar qué pines están abiertos.
```

File: docs/es/hardware_avr.md [hunk 15]
```diff
@@ -0,0 +1,181 @@
...
+
+Finalmente, puedes especificar la dirección en la que apuntan tus diodos. Esto puede `COL2ROW` o `ROW2COL`.
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
Finalmente, puedes especificar la dirección en la que apuntan tus diodos. Esto puede ser `COL2ROW` o `ROW2COL`.
```

File: docs/es/hardware_avr.md [hunk 16]
```diff
@@ -0,0 +1,181 @@
...
+
+```c
+#define DIODE_DIRECTION COL2ROW
+```
+
+#### Matriz de patas directas
+Para configurar un teclado en el que cada interruptor está conectado a una pata y un suelo separados en lugar de compartir las patas de fila y columna, uses `DIRECT_PINS`. La asignación define las patas de cada interruptor en filas y columnas, de izquierda a derecha. Debe ajustarse a los tamaños dentro de `MATRIX_ROWS` y `MATRIX_COLS`. Uses `NO_PIN` para rellenar espacios en blanco. Anula el comportamiento de `DIODE_DIRECTION`, `MATRIX_ROW_PINS` y `MATRIX_COL_PINS`.
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
Para configurar un teclado en el que cada interruptor está conectado a un pin y tierra separados en lugar de compartir los pines de fila y columna, usa `DIRECT_PINS`. La asignación define los pines de cada interruptor en filas y columnas, de izquierda a derecha. Debe ajustarse a los tamaños dentro de `MATRIX_ROWS` y `MATRIX_COLS`. Usa `NO_PIN` para rellenar espacios en blanco. Sobreescribe el comportamiento de `DIODE_DIRECTION`, `MATRIX_ROW_PINS` y `MATRIX_COL_PINS`.
```

File: docs/es/hardware_avr.md [hunk 17]
```diff
@@ -0,0 +1,181 @@
...
+}
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL */
+//#define DIODE_DIRECTION
+```
+
+### Configuración de retroiluminación
+
+QMK soporta retroiluminación en la mayoría de las patas GPIO. Algunos de ellas pueden ser manejados por el MCU en hardware. Para más detalles, consultes la [Documentación de Retroiluminación](feature_backlight.md).
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
QMK soporta retroiluminación en la mayoría de los pines GPIO. Algunos de ellos pueden ser manejados por el MCU en hardware. Para más detalles, consulta la [Documentación de Retroiluminación](feature_backlight.md).
```

File: docs/es/hardware_avr.md [hunk 18]
```diff
@@ -0,0 +1,181 @@
...
+#define BREATHING_PERIOD 6
+```
+
+### Otras opciones de configuración
+
+Hay un montón de características que se pueden configurar o ajustar en `config.h`. Debes consultar la página de [Opciones de Configuración](config_options.md) para más detalles.
+
+## `rules.mk`
+
+Usas el archivo `rules.mk` para decirle a QMK qué archivos construir y qué características habilitar. Si estás construyendo alrededor de un atmega32u4 debería puede dejar en gran parte estos valores predeterminados solos. Si estás usando otro MCU es posible que tengas que ajustar algunos parámetros.
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
Usa el archivo `rules.mk` para decirle a QMK qué archivos construir y qué características habilitar. Si estás construyendo sobre un atmega32u4 deberías poder dejar mayormente los valores predeterminados. Si estás usando otro MCU es posible que tengas que ajustar algunos parámetros.
```

File: docs/es/hardware_avr.md [hunk 19]
```diff
@@ -0,0 +1,181 @@
...
+
+### Opciones MCU
+
+Estas opciones le indican al sistema de compilación para qué CPU construir. Tengas mucho cuidado si cambias cualquiera de estos ajustes. Puedes inutilizar su teclado.
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
Estas opciones le indican al sistema de compilación para qué CPU construir. Ten mucho cuidado si cambias cualquiera de estos ajustes. Puedes inutilizar tu teclado.
```

File: docs/es/hardware_avr.md [hunk 20]
```diff
@@ -0,0 +1,181 @@
...
+MCU = atmega32u4
+F_CPU = 16000000
+ARCH = AVR8
+F_USB = $(F_CPU)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+```
+
+### Gestores de arranque
+
+El gestor de arranque es una sección especial de tu MCU que te permite actualizar el código almacenado en el MCU. Piensas en ello como una partición de aones para tu teclado.
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
El gestor de arranque es una sección especial de tu MCU que te permite actualizar el código almacenado en el MCU. Piensa en ello como una partición de rescate para tu teclado.
```

File: docs/es/hardware_avr.md [hunk 21]
```diff
@@ -0,0 +1,181 @@
...
+
+#### Ejemplo de gestor de arranque Pro Micro
+
+```make
+BOOTLOADER = caterina
+```
+
+### Opciones de construcción
+
+Hay un serie de características que pueden activar o desactivar en `rules.mk`. Consultes la página de [Opciones de Configuración](config_options.md#feature-options) para obtener una lista detallada y una descripción.
```
Comment (namelivia) [inline docs/es/hardware_avr.md]:
```suggestion
Hay un serie de características que se pueden activar o desactivar en `rules.mk`. Consulta la página de [Opciones de Configuración](config_options.md#feature-options) para obtener una lista detallada y una descripción.
```

File: docs/es/hardware_drivers.md [hunk 22]
```diff
@@ -0,0 +1,36 @@
+# Controladores de hardware QMK
+
+QMK se utiliza en un montón de hardware diferente. Mientras que el soporte para los  MCUs y las configuraciones de matriz más comunes está integrado, hay una serie de controladores que se pueden a un teclado para soportar hardware adicional. Los ejemplos incluyen ratones y otros dispositivos de apuntamiento, extensores de i/o para teclados divididos, modúlos Bluetooth, y pantallas LCD, OLED y TFT.
```
Comment (namelivia) [inline docs/es/hardware_drivers.md]:
```suggestion
QMK se utiliza en un montón de hardware diferente. Mientras que el soporte para los MCUs y las configuraciones de matriz más comunes está integrado, hay una serie de controladores que se pueden añadir para soportar hardware adicional al teclado. Los ejemplos incluyen ratones y otros dispositivos de apuntamiento, extensores de i/o para teclados divididos, modúlos Bluetooth, y pantallas LCD, OLED y TFT.
```

File: docs/es/hardware_drivers.md [hunk 23]
```diff
@@ -0,0 +1,36 @@
...
+
+# Descripción del sistema de controladores
+
+-->
+
+# Controladores disponibles
+
+## ProMicro (Solo AVR)
+
+Apoyo para direccionar patas en el ProMicro por su nombre Arduino en lugar de su nombre AVR. Esto necesita ser mejor documentado. Si estás tratando de hacer esto y leer el código no ayuda por favor [abrir un problema](https://github.com/qmk/qmk_firmware/issues/new) y podemos ayudarte por el proceso.
```
Comment (namelivia) [inline docs/es/hardware_drivers.md]:
```suggestion
Soporte para direccionar pines en el ProMicro por su nombre Arduino en lugar de su nombre AVR. Esto necesita ser mejor documentado. Si estás tratando de hacer esto y leer el código no ayuda por favor [abre una issue](https://github.com/qmk/qmk_firmware/issues/new) y podemos ayudarte por el proceso.
```

File: docs/es/hardware_drivers.md [hunk 24]
```diff
@@ -0,0 +1,36 @@
...
+
+## Controlador OLED SSD1306
+
+Apoyo para pantallas OLED basadas en SSD1306. Para obtener más información consultes la página de [Característica de Controlador OLED](feature_oled_driver.md).
```
Comment (namelivia) [inline docs/es/hardware_drivers.md]:
```suggestion
Srte para pantallas OLED basadas en SSD1306. Para obtener más información consulta la página de [Característica de Controlador OLED](feature_oled_driver.md).
```

File: docs/es/hardware_drivers.md [hunk 25]
```diff
@@ -0,0 +1,36 @@
...
+
+## uGFX
+
+Puedes hacer uso de uGFX dentro de QMK para manejar LCDs caracteres y gráficos, matrices de LED, OLED, TFT, y otras tecnologías de visualización. Esto necesita ser mejor documentado. Si estás tratando de hacer esto y leer el código no ayuda por favor [abrir un problema](https://github.com/qmk/qmk_firmware/issues/new) y podemos ayudarte por el proceso.
```
Comment (namelivia) [inline docs/es/hardware_drivers.md]:
```suggestion
Puedes hacer uso de uGFX dentro de QMK para manejar LCDs de caracteres y gráficos, matrices de LED, OLED, TFT, y otras tecnologías de visualización. Esto necesita ser mejor documentado. Si estás tratando de hacer esto y leer el código no ayuda por favor [abre una issue](https://github.com/qmk/qmk_firmware/issues/new) y podemos ayudarte por el proceso.
```

File: docs/es/hardware_drivers.md [hunk 26]
```diff
@@ -0,0 +1,36 @@
...
+
+## WS2812 (Solo AVR)
+
+Apoyo para LEDs WS2811/WS2812{a,b,c}. Para obtener más información consultes la página de [Luz RGB](feature_rgblight.md).
```
Comment (namelivia) [inline docs/es/hardware_drivers.md]:
```suggestion
Soporte para LEDs WS2811/WS2812{a,b,c}. Para obtener más información consulta la página de [Luz RGB](feature_rgblight.md).
```

File: docs/es/hardware_drivers.md [hunk 27]
```diff
@@ -0,0 +1,36 @@
...
+
+## IS31FL3731
+
+Apoyo para hasta 2 controladores. Cada controlador implementa 2 matrices charlieplex para direccionar LEDs individualmente usando I2C. Esto permite hasta 144 LEDs del mismo color o 32 LEDs RGB. Para obtener más información sobre cómo configurar el controlador, consultes la página de [Matriz RGB](feature_rgb_matrix.md).
```
Comment (namelivia) [inline docs/es/hardware_drivers.md]:
```suggestion
Soporte para hasta 2 controladores. Cada controlador implementa 2 matrices charlieplex para direccionar LEDs individualmente usando I2C. Esto permite hasta 144 LEDs del mismo color o 32 LEDs RGB. Para obtener más información sobre cómo configurar el controlador, consulta la página de [Matriz RGB](feature_rgb_matrix.md).
```

File: docs/es/hardware_drivers.md [hunk 28]
```diff
@@ -0,0 +1,36 @@
...
+
+## IS31FL3733
+
+Apoyo para hasta un solo controlador con espacio para expansión. Cada controlador puede controlar 192 LEDs individuales o 64 LEDs RGB. Para obtener más información sobre cómo configurar el controlador, consultes la página de [Matriz RGB](feature_rgb_matrix.md).
```
Comment (namelivia) [inline docs/es/hardware_drivers.md]:
```suggestion
Soporte para hasta un solo controlador con espacio para expansión. Cada controlador puede controlar 192 LEDs individuales o 64 LEDs RGB. Para obtener más información sobre cómo configurar el controlador, consulta la página de [Matriz RGB](feature_rgb_matrix.md).
```

File: docs/es/hardware_keyboard_guidelines.md [hunk 29]
```diff
@@ -0,0 +1,149 @@
+# Pautas del teclado QMK
+
+Desde sus inicios, QMK ha crecido a pasos agigantados gracias a personas como tú que contribuyes a la creación y mantenimiento de nuestros teclados comunitarios. A medida que hemos crecido hemos descubierto algunos patrones que funcionan bien, y pedimos que ajusta a ellos para que sea más fácil para que otras personas se beneficien de tu duro trabajo.
```
Comment (namelivia) [inline docs/es/hardware_keyboard_guidelines.md]:
```suggestion
Desde sus inicios, QMK ha crecido a pasos agigantados gracias a personas como tú que contribuyes a la creación y mantenimiento de nuestros teclados comunitarios. A medida que hemos crecido hemos descubierto algunos patrones que funcionan bien, y pedimos que te ajustes a ellos para que sea más fácil para que otras personas se beneficien de tu duro trabajo.
```

File: docs/es/hardware_keyboard_guidelines.md [hunk 30]
```diff
@@ -0,0 +1,149 @@
...
+
+* `412_64`
+* `chimera_ortho`
+* `clueboard/66/rev3`
+* `planck`
+* `v60_type_r`
+
+## Subcarpetas
+
+QMK utiliza subcarpetas tanto para organización como para compartir código entre las revisiones del mismo teclado. Puede anidar carpetas hasta 4 niveles de profundidad:
```
Comment (namelivia) [inline docs/es/hardware_keyboard_guidelines.md]:
```suggestion
QMK utiliza subcarpetas tanto para organización como para compartir código entre las revisiones del mismo teclado. Puedes anidar carpetas hasta 4 niveles de profundidad:
```

File: docs/es/hardware_keyboard_guidelines.md [hunk 31]
```diff
@@ -0,0 +1,149 @@
...
+* `readme.md`
+* `info.json`
+* `config.h`
+* `rules.mk`
+* `<keyboard_name>.c`
+* `<keyboard_name>.h`
+
+### `readme.md`
+
+Todos los proyectos necesitan tener un archivo `readme.md` que explica lo que es el teclado, quién lo hizo y dónde está disponible. Si es aplicable, también debe contener enlaces a más información, como el sitio web del fabricante. Por favor, siga la [plantilla publicada](documentation_templates.md#keyboard-readmemd-template).
```
Comment (namelivia) [inline docs/es/hardware_keyboard_guidelines.md]:
```suggestion
Todos los proyectos necesitan tener un archivo `readme.md` que explica lo que es el teclado, quién lo hizo y dónde está disponible. Si es aplicable, también debe contener enlaces a más información, como el sitio web del fabricante. Por favor, sigue la [plantilla publicada](documentation_templates.md#keyboard-readmemd-template).
```

File: docs/es/hardware_keyboard_guidelines.md [hunk 32]
```diff
@@ -0,0 +1,149 @@
...
+
+### `info.json`
+
+Este archivo es utilizado por la [API de QMK](https://github.com/qmk/qmk_api). Contiene la información [configurador de QMK](https://config.qmk.fm/) necesita mostrar una representación de su teclado. También puede establecer metadatos aquí. Para más información, consulte la [página de referencia](reference_info_json.md).
```
Comment (namelivia) [inline docs/es/hardware_keyboard_guidelines.md]:
```suggestion
Este archivo es utilizado por la [API de QMK](https://github.com/qmk/qmk_api). Contiene la información que[configurador de QMK](https://config.qmk.fm/) necesita mostrar en una representación de su teclado. También puede establecer metadatos aquí. Para más información, consulta la [página de referencia](reference_info_json.md).
```

File: docs/es/hardware_keyboard_guidelines.md [hunk 33]
```diff
@@ -0,0 +1,149 @@
...
+
+### `config.h`
+
+Todos los proyectos necesitan tener un archivo `config.h` que establece las cosas como el tamaño de la matriz, nombre del producto, USB VID/PID, descripción y otros ajustes. En general, uses este archivo para establecer la información esencial y los valores predeterminados para tu teclado que siempre funcionará.
```
Comment (namelivia) [inline docs/es/hardware_keyboard_guidelines.md]:
```suggestion
Todos los proyectos necesitan tener un archivo `config.h` que establece cosas como el tamaño de la matriz, nombre del producto, USB VID/PID, descripción y otros ajustes. En general, usa este archivo para establecer la información esencial y los valores predeterminados para tu teclado que siempre funcionarán.
```

File: docs/es/hardware_keyboard_guidelines.md [hunk 34]
```diff
@@ -0,0 +1,149 @@
...
+
+### `rules.mk`
+
+La presencia de este archivo significa que la carpeta es un destino de teclado y se puede utilizar en las órdenes `make`. Aquí es donde te estableces el entorno de compilación para tu teclado y te configuras el conjunto predeterminado de características.
```
Comment (namelivia) [inline docs/es/hardware_keyboard_guidelines.md]:
```suggestion
La presencia de este archivo indica que la carpeta es un destino de teclado y se puede utilizar en las órdenes `make`. Aquí es donde estableces el entorno de compilación para tu teclado y configuras el conjunto predeterminado de características.
```

File: docs/es/hardware_keyboard_guidelines.md [hunk 35]
```diff
@@ -0,0 +1,149 @@
...
+|-------------|-------------|-------------|
+| LAYOUT_all | default | Un diseño que soporta tanto ISO como ANSI |
+| LAYOUT_ansi | default_ansi | Un diseño ANSI |
+| LAYOUT_iso | default_iso | Un diseño ISO |
+
+## Archivos de Imagen/Hardware
+
+En un esfuerzo por mantener el tamaño de repo abajo ya no estamos aceptando archivos binarios de cualquier formato, con pocas excepciones. Alojarlos en otro lugar (por ejemplo <https://imgur.com>) y enlazarlos en el `readme.md` es preferible.
+
+Archivos de hardware (tales como placas, casos, pcb) se puede contribuir a la [qmk.fm repo](https://github.com/qmk/qmk.fm) y estarán disponibles en [qmk.fm](http://qmk.fm). Archivos descargables se almacenan en `/<teclado>/` (nombre sigue el mismo formato que el anterior) que se sirven en `http://qmk.fm/<teclado>/`, y se generan las páginas de `/_pages/<teclado>/` que se sirven en la misma ubicación (archivos .md se generan en archivos .html por Jekyll). Echa un vistazo a la carpeta` lets_s ' para un ejemplo.
```
Comment (namelivia) [inline docs/es/hardware_keyboard_guidelines.md]:
```suggestion
Para archivos de hardware (tales como placas, casos, pcb) puedes contribuir a [qmk.fm repo](https://github.com/qmk/qmk.fm) y estarán disponibles en [qmk.fm](http://qmk.fm). Archivos descargables se almacenan en `/<teclado>/` (nombre sigue el mismo formato que el anterior), se sirven en `http://qmk.fm/<teclado>/`, y se generan páginas de `/_pages/<teclado>/` que se sirven en la misma ubicación (Los archivos .md se transforman en archivos .html mediante Jekyll). Echa un vistazo a la carpeta` lets_s ' para ver un ejemplo.
```

File: docs/es/hardware_keyboard_guidelines.md [hunk 36]
```diff
@@ -0,0 +1,149 @@
...
+|-------------|-------------|-------------|
+| LAYOUT_all | default | Un diseño que soporta tanto ISO como ANSI |
+| LAYOUT_ansi | default_ansi | Un diseño ANSI |
+| LAYOUT_iso | default_iso | Un diseño ISO |
+
+## Archivos de Imagen/Hardware
+
+En un esfuerzo por mantener el tamaño de repo abajo ya no estamos aceptando archivos binarios de cualquier formato, con pocas excepciones. Alojarlos en otro lugar (por ejemplo <https://imgur.com>) y enlazarlos en el `readme.md` es preferible.
+
+Para archivos de hardware (tales como placas, casos, pcb) puedes contribuir a [qmk.fm repo](https://github.com/qmk/qmk.fm) y estarán disponibles en [qmk.fm](http://qmk.fm). Archivos descargables se almacenan en `/<teclado>/` (nombre sigue el mismo formato que el anterior), se sirven en `http://qmk.fm/<teclado>/`, y se generan páginas de `/_pages/<teclado>/` que se sirven en la misma ubicación (Los archivos .md se generan en archivos .html mediante Jekyll). Echa un vistazo a la carpeta` lets_s ' para ver un ejemplo.
```
Comment (noroadsleft) [inline docs/es/hardware_keyboard_guidelines.md]: Correcting the `lets_split` folder name here.

```suggestion
Para archivos de hardware (tales como placas, casos, pcb) puedes contribuir a [qmk.fm repo](https://github.com/qmk/qmk.fm) y estarán disponibles en [qmk.fm](http://qmk.fm). Archivos descargables se almacenan en `/<teclado>/` (nombre sigue el mismo formato que el anterior), se sirven en `http://qmk.fm/<teclado>/`, y se generan páginas de `/_pages/<teclado>/` que se sirven en la misma ubicación (Los archivos .md se generan en archivos .html mediante Jekyll). Echa un vistazo a la carpeta `lets_split` para ver un ejemplo.
```

**Comment by noroadsleft** [comment]
I reviewed both @kbravh and @namelivia's reviews (so meta :grinning:) and verified that their suggestions were applied in https://github.com/qmk/qmk_firmware/pull/7161/commits/68f177521864fb6c4d04ae25164cc6f609fc20e3, save for [this comment](https://github.com/qmk/qmk_firmware/pull/7161#discussion_r341017395). My one suggestion is on the same line.

**Pull request merged**